app/home/component/CardList/CardList.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { Stack, initializeIcons } from "@fluentui/react";
+import React, { useCallback, useEffect } from "react";
+
+import { CardType, Card } from "../Card/Card";
+
+import { convertMovieResponseToCard } from "../utils";
+
+import { fetchPopularMovies } from "@/app/lib/http/movie/movie.service";
+
+export type CardListProps = {
+  cards: CardType[];
+  currentPage: number;
+};
+
+export const CardList: React.FC<CardListProps> = (props) => {
```

Comment:
Don't assign a single variable as props. Decomposing it should be standard unless you have a really good reason not to do so. It just harms the readability of the code and you won't be able to tell if all the nested values under this `props` have been used or not.

Reply:
**In your way, you can't tell "yay" hasn't been used in the component easily**
![Screenshot 2023-10-22 at 12 50 10 AM](https://github.com/koh1project/movies/assets/44686790/4a595fde-5a49-4e07-9ad9-681d4a431d43)

**Decomposing. You can tell "yay" hasn't been used in the component right away**
![Screenshot 2023-10-22 at 12 50 53 AM](https://github.com/koh1project/movies/assets/44686790/fa98eced-2219-4557-b936-dc1983f7c0ff)

Reply:
Updated the parameter declaration with destructuring.